requirements/common.txt

```diff
@@ -10,3 +10,4 @@ numpy; sys_platform != 'win32'
 numpy==1.19.3; sys_platform == 'win32' #see https://tinyurl.com/y3dm3h86
 nvgpu; sys_platform != 'win32'
 nvgpu==0.8.0; sys_platform == 'win32'
+pyyaml
```

Comment:
Should we tie version to latest stable release, say `6.0`?

Reply:
@maaquib please confirm if python metrics requests specific pyyaml version. Thanks!

Reply:
There is no requirement for a specific version. We can pin in to the latest one.